README.md

```diff
@@ -1,8 +1,20 @@
 # altinn-components
-This package is a set of reusable components for building web applications for [Altinn 3](https://docs.altinn.studio/nb/community/about/).
-This is *not* a general purpose library, but a library of components that are specifically designed for the Altinn platform, to ensure
-a consistent look and feel across all applications. It also contains a storybook with stories explaining the purpose and usage of each component, and
-with real examples of how to use them separately or together (composition).
+
+Altinn's frontend is developed by multiple product teams, all working toward the shared goal of delivering a seamless, unified experience—ensuring users perceive it as a single and robust product. 
+This package represents a collaborative effort between designers and developers across these teams, consolidating solutions to shared challenges. 
+It provides Altinn-specific React components tailored to address the common UI needs of the various teams within Altinn.   
+
+Today these teams are [Team Arbeidsflate](https://github.com/digdir/dialogporten-frontend), [Team authorization](https://github.com/Altinn/altinn-access-management-frontend), and Team Portal.
+
+Note that this package is specifically designed to meet the unique needs of Altinn's product teams and is not intended to serve as a general-purpose library. 
```

Comment:
```suggestion
Note that this package is specifically designed to meet the unique needs of the aforementioned teams and is not intended to serve as a general-purpose library. 
```
"Altinn product teams" include teams like Apps, Studio, AppsMigration, etc. We should be weary of implying that this is inteded for them as well.